scenes/quests/lore_quests/quest_001/quest.tres

```diff
@@ -0,0 +1,15 @@
+[gd_resource type="Resource" script_class="Quest" load_steps=3 format=3 uid="uid://doovydomib7rj"]
+
+[ext_resource type="Script" uid="uid://dts1hwdy3phin" path="res://scenes/menus/storybook/components/quest.gd" id="1_3ntet"]
+[ext_resource type="SpriteFrames" uid="uid://clx8sssi4tw8q" path="res://scenes/game_elements/characters/shared_components/sprite_frames/musician.tres" id="2_xjg3h"]
+
+[resource]
+script = ExtResource("1_3ntet")
+title = "The Musician's Quest"
+description = "StoryWeaver meets a musician and a series of terrifying creatures."
```

Comment:
I guess @Stregoica777 or @hydrolet would like to review this? I love it.

Reply:
Yes, the title and description are meant as placeholders to revise later, but good enough to not look broken.